CHANGELOG.md

```diff
@@ -8,6 +8,12 @@
   * Methods that set a single value will now return the new value as the resolution of the returned promise e.g. `r.toggle('foo')` if `foo` is falsey will return a promise that resolves to `true`. `add`/`subtract`/`set` behave similarly.
   * Decorators and custom events can now access their local context, in the case of a yielded attribute partial, with `this.getLocalContext()`.
 
+# 1.2.10, 1.3.13
```

Comment:
Not a big deal at all, but I usually put an (unreleased) on there for versions that haven't been pushed to their release branch yet.

Reply:
Got it! I'll do it next time.